io.enpass.Enpass.yaml

```diff
@@ -0,0 +1,95 @@
+# io.enpass.Enpass.yaml
+app-id: io.enpass.Enpass
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: enpass
+tags: [proprietary]
+separate-locales: false
+finish-args:
+    # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+    # Notification access
+  - --talk-name=org.freedesktop.Notifications
+    # System Tray Icon
+  - --talk-name=com.kde.StatusNotifierWatcher
+    # Secret Service
+  - --own-name=org.freedesktop.secrets
+    # Favicon download
+  - --share=network
+    # Required to save vaults in default location <$HOME/Documents>
+  - --filesystem=xdg-documents
+modules:
+  - name: enpass
+    buildsystem: simple
+    build-options:
+      env:
+        FLATPAK_ID: io.enpass.Enpass
+    build-commands:
+      - |
+        # Install scripts
+        install -Dm 755 apply_extra --target-directory ${FLATPAK_DEST}/bin/
+        install -Dm 755 enpass --target-directory ${FLATPAK_DEST}/bin/
+
+        # Install Desktop, MetaInfo, and MIME
+        install -Dm 644 ${FLATPAK_ID}.desktop --target-directory ${FLATPAK_DEST}/share/applications/
+        install -Dm 644 ${FLATPAK_ID}.metainfo.xml --target-directory ${FLATPAK_DEST}/share/metainfo/
+        install -Dm 644 ${FLATPAK_ID}.mime.xml --target-directory ${FLATPAK_DEST}/share/mime/packages/
+
+        # Install icons
```

Comment:
the SVG and the PNG still look different. They shouldn't. You could also just have the SVG, that all that is needed.

Reply:
[Removed PNG and colorized SVG with Inkscape](https://github.com/ShobuPrime/flathub/commit/7c5511e7bced4827605700be6d445510953d03cf#diff-4e1d2b12a1a9cb4a6aa05390bf69c7168008557903f720b62b5447c3b1ef7018L41) :heavy_check_mark: